unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Settings/PlayerPrefsGeneralSettingsRepository.cs

```diff
@@ -59,6 +61,9 @@ public void Save()
             settingsByKey.SetBool(INVERT_Y_AXIS, currentSettings.invertYAxis);
             settingsByKey.SetFloat(SKYBOX_TIME, currentSettings.skyboxTime);
             settingsByKey.SetFloat(FIRST_PERSON_CAMERA_FOV, currentSettings.firstPersonCameraFOV);
+            settingsByKey.SetBool(HIDE_UI, currentSettings.hideUI);
+            settingsByKey.SetBool(SHOW_AVATAR_NAMES, currentSettings.showAvatarNames);
+            settingsByKey.SetBool(CAMERA_MOVEMENT_MODE, currentSettings.leftMouseButtonCursorLock);
```

Comment:
**Question**: Have you confirmed that it is a requirement to store these 3 settings (`HIDE_UI`, `SHOW_AVATAR_NAMES` and `CAMERA_MOVEMENT_MODE`)? It sounds to me like we were not saving some of them on purpose. For instance, for me it's a normal behavior that the "HIDE UI" feature always initializes deactivated as default at the beginning.

Before applying the change, I would ask product to be sure.

Reply:
Good point, @sandrade-dcl. `CAMERA_MOVEMENT_MODE` is new feature, so it is supposed to be saved. I We also agreed with @eordano, that `SHOW_AVATAR_NAMES ` should be saved as well (imagine someone who don't like it - he will need to go to settings on every enter to disable it). Thus, only `HIDE_UI` will not be saved. I will push fix soon.